Makefile.test

```diff
@@ -35,7 +35,7 @@ run-tests-fota: build-test-fota-byoc
 	find ./build-test/bin -name "*-test" | xargs -n1 -I{} sh -c '{} || exit 255'
 
 run-tests-with-valgrind: build-test-byoc
-	find ./build-test/bin -name "*-test" | xargs -n1 -I{} sh -c 'valgrind --track-origins=yes --num-callers=100 --leak-check=full --show-reachable=yes --suppressions=./valgrind-unit-tests.supp --error-exitcode=255 {} || exit 255'
+	find ./build-test/bin -name "*-test" | xargs -n1 -I{} sh -c 'valgrind --track-origins=yes --num-callers=100 --leak-check=full --show-reachable=yes --error-exitcode=255 {} || exit 255'
```

Comment:
Wouldn't removing suppression generate issues?

Reply:
You should remove the --show-reachable=yes flag too.

Reply:
what is the reason to force the exit code to be 255 ?

Reply:
removed show reachable flag. I don't know the reason of the exit code, hence removed that as well.